dialogue-annotations-example/src/main/java/com/palantir/myservice/example/IHateExceptionsErrorDecoder.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.myservice.example;
+
+import com.palantir.dialogue.Response;
+import com.palantir.dialogue.annotations.ErrorDecoder;
+
+public final class IHateExceptionsErrorDecoder implements ErrorDecoder {
```

Comment:
This is a pretty crude name, can we change it to something more appropriate for a public open-source library?

Reply:
Ack

Reply:
https://github.com/palantir/dialogue/pull/1269